external/js/cothority/src/darc/identity-wrapper.ts

```diff
@@ -24,3 +24,35 @@ export default class IdentityWrapper extends Message<IdentityWrapper> {
     return Buffer.from([]);
   }
 }
+
+/**
+ * Identitiy is an abstract class for all the Darcs's identities
```

Comment:
```suggestion
 * Identity is an abstract class for all the Darcs's identities
```

Search... I think this typo is other places maybe.